beta/README.md

```diff
@@ -1,5 +1,121 @@
+  - Deployments or updates _must_ not fail for inexplicable reasons.
+- Documentation
+  - Documentation for the primary anticipated use case _must_ be complete, though it need not be in it's final form since it is anticipated that documentation will change while in beta. Complete documentation, in this context, means the following are included:
+    - Complete instructions for the most common deployment configuration this example targets.
+    - Complete instructions for updating the SCIM bridge.
+    - Terminal commands, if applicable, _must_ be provided for both Unix shells and Powershell, where both are supported.
+  - Documentation for less common use-cases _may_ be incomplete. If incomplete documentation is provided, it _must_ be noted as incomplete in an obvious way to the reader.
+  - Documentation _must_ indicate approximate provisioning capacity of a SCIM bridge deployed in the documented manner. Documentation for scaling the SCIM bridge is _not_ required.
+  - Documentation _should_ conform to the [1Password Style Guide](https://support.1password.com/style-guide/).
+
+### Process for publishing a beta deployment example
+
+- Review the criteria above to ensure your proposed deployment example meets the criteria. 
+- Create a PR containing the deployment example.
+  - Documentation and all configuration files should be contained in `./beta/<new deployment method name>/`
+  - In the PR, briefly describe the rationale for the proposed deployment (e.g., how it improves on existing methods, or how it addresses a use-case not already served).
+  - Describe anything that may still need to be implemented (e.g., Google Workspace support, certain vertical scaling, etc).
+  - Call out any specific tests or scenarios that you would like the reviewer to focus on. 
+- Update README.md at the root of the repository
+  - Add the deployment method to the list of beta deployments in [`README.md`](../README.md#beta-deployment) in the following format:   
+  `- ✨ **NEW** [Deployment Name](/beta/deployment-name)`
+- If you are a 1Password employee, assign reviewers from the Solutions Architect or AOP developer teams. 
+  - If you are a community member, a 1Password employee will review your PR when time and resources permit.
+- The beta deployment example will be merged into Main when:
+  - The reviewer(s) have tested the example successfully according to the use case it's designed to address.
+  - The documentation conforms to the criteria above. 
+  - All outstanding changes requested by reviewers have been addressed in a way that satisfies the reviewer. 
+  - The reviewer has, in their sole discretion, determined that the example is worth including in the repository, regardless of how well the example meets the criteria above. 
+
+> **Note**  
+> 1Password, in it's sole discretion, may choose to not publish examples for _any reason_, even if the example meets or exceeds all of the above criteria. 
```

Comment:
its is the possessive, it's is the contraction (switch to its)

Reply:
🤦